crates/preset_env_base/src/query.rs

```diff
@@ -96,8 +96,31 @@ impl Query {
     }
 }
 
-pub fn targets_to_versions(v: Option<Targets>) -> Result<Versions, Error> {
+pub fn targets_to_versions(v: Option<Targets>, path: Option<PathBuf>) -> Result<Versions, Error> {
     match v {
+        #[cfg(not(target_arch = "wasm32"))]
+        None => {
+            let mut browserslist_opts = browserslist::Opts::new()
+                .mobile_to_desktop(true)
+                .ignore_unknown_versions(true)
+                .to_owned();
+            if let Some(path) = path {
+                browserslist_opts.path(
+                    path.clone()
+                        .into_os_string()
+                        .to_str()
+                        .unwrap_or_else(|| panic!("Invalid path \"{:?}\"", path)),
+                );
+            }
+            let distribs = browserslist::execute(&browserslist_opts)
```

Comment:
Does this work with configs that execute javascript? cc @g-plane

Reply:
No, it doesn't execute any code.